packages/compartment-mapper/src/node-modules.js

```diff
@@ -395,6 +396,40 @@ const graphPackages = async (
   return graph;
 };
 
+/**
+ * Compute the lexically shortest path from the entry package to each
+ * transitive dependency package.
+ * The path is a delimited with hashes, so hash is forbidden to dependency
+ * names.
+ * The empty string is a sentinel for a path that has not been computed.
+ *
+ * The shortest path serves as a suitable sort key for generating archives that
+ * are consistent even when the package layout on disk changes, as the package
+ * layout tends to differ between installation with and without devopment-time
+ * dependencies.
+ *
+ * @param {Graph} graph
+ * @param {string} location
+ * @param {string} path
+ */
+const trace = (graph, location, path) => {
+  const node = graph[location];
+  if (node.path !== '' && node.path <= path) {
```

Comment:
This is not how 'a'a compares paths, at least not anymore.
The preference is for human consumption so
zbzb/x is better than a/b/x

Not sure how relevant that is here. I understand this is only meant to aid the sort by label to avoid producing different results each time.